bin/pygrb/pycbc_make_offline_grb_workflow

```diff
@@ -248,11 +248,31 @@ else:
     IPN = False
 
 # Get bank_veto_bank.xml if running bank veto
-# TODO: Finalize config file location of bank veto bank
 if wflow.cp.has_option('workflow-inspiral', 'bank-veto-bank-file'):
     bank_veto_file_path = wflow.cp.get('workflow-inspiral', 'bank-veto-bank-file')
```

Comment:
Do this (on `bank_veto_file_path`):

https://github.com/gwastro/pycbc/blob/master/bin/workflows/pycbc_make_uberbank_workflow#L162

it will convert it to a File object (assuming there is only one file). It will work "properly" if the file is in CVMFS, and can pull files from the github repo.